Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -874,6 +874,14 @@ public IMixedRealityInputSource RequestNewGenericInputSource(string name, IMixed
             return new BaseGenericInputSource(name, pointers, sourceType);
         }
 
+        /// <inheritdoc />
+        public BaseGlobalInputSource RequestNewGlobalInputSource(string name, IMixedRealityFocusProvider focusProvider = null, InputSourceType sourceType = InputSourceType.Other)
```

Comment:
I personally would default to the InputSystem's Focus Provider, though I think it hides details about how we intend this input source to be used, since the input source's constructor expects a focus provider.